documentation/en/source/pages/command-line/migration.rst

```diff
@@ -18,4 +18,15 @@ Sub Commands
 ``list`` sub-command
 ~~~~~~~~~~~~~~~~~~~~
 
-Displays all the migrations for the user.
\ No newline at end of file
+Displays all the migrations for the user.
+
+
+``launch`` sub-command
+~~~~~~~~~~~~~~~~~~~~
+
+Launches a new migration.
+
+	* ``--file`` (mandatory): json or yaml file providing the migration configuration
+	* ``--migration-port`` (optional): the ssh port of the running system to migrate
+	* ``--migration-password`` (optional): the root password to authenticate to the running system to migrate
+	* ``--dir`` (optional): the directory where to install the uforge-migrate.bin binary used to launch the migration
```

Comment:
do we really need those parameters?

Reply:
It was to keep coherency with scan run command.

About port and password, I agree because they are optional in the file so:
- if port is not in the file, we use 22 as default
- if password is not in the file, we ask for giving it after launching the command.

About dir, it's not in the file so this was to give to the user the possibility to give the "upload binary directory" in the source. If not given, use /tmp